project/src/main/java/com/google/sps/servlets/CreateProductServlet.java

```diff
@@ -64,18 +72,35 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String productCategory = request.getParameter("productCategory");
     String businessId = userService.getCurrentUser().getUserId();
 
-    // TODO: catch any exceptions for string to float conversion.
-    float price = Float.parseFloat(request.getParameter("price"));
+    float price;
+    try {
+      price = Float.parseFloat(request.getParameter("price"));
+    } catch (NumberFormatException e) {
+      price = 0.0f;
```

Comment:
Maria mentioned to me to output the exception so you know what it  is. Idk if that applies here though, because you might already know.